Cargo.toml

```diff
@@ -15,7 +15,7 @@ categories = ["web-programming::http-client", "no-std", "parsing", "asynchronous
 
 [dependencies]
 eventsource-stream = "0.2.3"
-reqwest = { version = "0.11", default-features = false, features = ["stream"] }
+reqwest = { version = "0.11", path = "../reqwest", default-features = false, features = ["stream"] }
```

Comment:
Is the `path` value expected ?

`cargo check` fail (see below for the error message) when until I remove that field.

```txt
    Updating git repository `https://github.com/tmpfs/reqwest-eventsource.git`
error: no matching package named `reqwest` found
location searched: https://github.com/tmpfs/reqwest-eventsource.git?rev=6ce0923ccc7cae4ce39a600b35425a62edc632b5#6ce0923c
required by package `reqwest-eventsource v0.4.0 (https://github.com/tmpfs/reqwest-eventsource.git?rev=6ce0923ccc7cae4ce39a600b35425a62edc632b5#6ce0923c)`
    ... which satisfies git dependency `reqwest-eventsource` of package `sse-web v0.1.0 (/home/[Redacted]/Documents/platform_sse/sse-web)`
```